### DISCUSSION TITLE
License purchased issue

### DISCUSSION THREAD

**Discussion by Embeddedwizard** [aclap-dev/video-downloadhelper#1517] [License & Purchase requests]
### What's your issue?

I have purchased the license and its working on my browser. for some reason i want reset the laptop and use new browser 
but when i install the license its not getting verified or not able to install it on new browser 

how to remove old license from the browser and link it to new browser ?

### The 10 first digits (only 10) of your license number or email address

kannan.dhanuz@gmail.com

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You don't need to remove it, you can use your license on all your devices: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
